composer.json

```diff
@@ -16,29 +16,67 @@
             "homepage": "http://www.aoe.com"
         }
     ],
-    "config": {
-        "secure-http": false
-    },
     "require": {
-        "php": ">=5.1.0",
+        "php": ">=5.5.0",
         "typo3/cms-core": ">=6.2.0,<=7.6.99"
     },
     "require-dev": {
-        "typo3/cms": "^6.2",
-        "phpunit/phpunit": "4.8.*",
+        "typo3/cms": "^7.6",
+        "nimut/testing-framework": "^2.0.0",
         "phpunit/phpcov": "2.0.*",
         "squizlabs/php_codesniffer": "2.3.*",
         "sebastian/phpcpd": "2.0.*",
         "phpmd/phpmd": "2.2.*"
     },
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://composer.typo3.org/"
+        }
+    ],
     "autoload": {
-        "classmap": [
-            "Classes"
-        ]
+        "psr-4": {
+            "Aoe\\TruncateJob\\": "Classes/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Aoe\\TruncateJob\\Tests\\": "Tests",
+            "TYPO3\\CMS\\Core\\Tests\\": ".Build/vendor/typo3/cms/typo3/sysext/core/Tests/"
```

Comment:
While using Nimut testing framework we don't need to configure autoloading of core test files. This is handled by Nimut.